pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -486,6 +486,12 @@ func (h *NetworkUtilsHandler) BindTapDeviceToBridge(tapName string, bridgeName s
 		return fmt.Errorf("failed to set tap device %s up; %v", tapName, err)
 	}
 
+	// turn TX offload checksum because it causes dhcp failures
```

Comment:
Why is it in the `BindTapDeviceToBridge` method. It is confusing.

Reply:
Where would you have it ? 

On the BindMechanism `preparePodNetworkInterfaces` method (each bind mechanism would need to do it) or in the BindMechanism `createBridge` method ? 
  - [preparePodNetworkInterfaces](https://github.com/kubevirt/kubevirt/blob/ff7494b22c57123469328129e1c6d0b6f4e8a9aa/pkg/virt-launcher/virtwrap/network/podinterface.go#L749)
  - [createBridge](https://github.com/kubevirt/kubevirt/blob/ff7494b22c57123469328129e1c6d0b6f4e8a9aa/pkg/virt-launcher/virtwrap/network/podinterface.go#L854)

Reply:
`createBridge` sound like a good place.